package.json

```diff
@@ -5,7 +5,10 @@
   "main": "lib/Slider",
   "jsnext:main": "src/Slider.jsx",
   "scripts": {
-    "build": "npm run clean && babel src -d lib",
+    "prebuild": "npm run clean",
+    "build": "npm run clean && babel src -d lib && npm run build:css -- --optimize ",
```

Comment:
the clean script can be removed here, since it's in prebuild.

also, let's add a "postbuild" that does `npm run build:css`, instead of doing it inside the build step?

Reply:
Isn't the css building part of the build step (just like, semantically). 

Maybe let's move the `babel src -d lib` to a `npm run build:js` and then change `npm run build` to be `npm run build:js && npm run build:css -- --optimize`

Reply:
oh sure that's fine too